pkg/runtime/reconciler.go

```diff
@@ -303,54 +303,33 @@ func (r *resourceReconciler) handleCacheError(
 	return r.HandleReconcileError(ctx, desired, latest, requeue.NeededAfter(err, roleARNNotAvailableRequeueDelay))
 }
 
-func (r *resourceReconciler) handleAdoption(
-	ctx context.Context,
-	rm acktypes.AWSResourceManager,
-	desired acktypes.AWSResource,
+func (r *resourceReconciler) handlePopulation(
```

Comment:
interesting rename, can you please add a comment explaining what is the purpose (and non-purpose) of this function?